### PR TITLE
Phones from api

### PR DESCRIPTION
**Pull request by OlhaSheliakina**

### CODE REVIEW

**Review by perfectAcedia — APPROVED** [review]

**Review by NazarBaraban — APPROVED** [review]

File: src/pages/CatalogPage.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import { Dropdown } from '../components/Dropdown';
 import { Card } from '../components/Card';
-import axios from 'axios';
+// import axios from 'axios';
```
Comment (olena-ole) [inline src/pages/CatalogPage.tsx:4]: delete comment

**Review by olena-ole — DISMISSED** [review]
GJ! Just delete the comment

File: src/redux/api/product.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    baseUrl: 'https://dreamteam.onrender.com',
+  }),
+  endpoints: (builder) => ({
+    getProducts: builder.query({
+      query: () => `/products`,
+    }),
+  }),
+});
+
+export const { useGetProductsQuery } = productApi;
```
Comment (MaxSchmide) [inline src/redux/api/product.ts:14]: rename file to your func

File: src/pages/CatalogPage.tsx [hunk 3]
```diff
@@ -38,35 +39,33 @@ const CatalogPage: React.FC = () => {
...
-  // #endregion
+  const {
+    data: products,
+    isLoading,
+    isError,
+  } = useGetProductsQuery({
+    perPage,
+    page: currentPage,
+    sortBy,
+  });
```
Comment (MaxSchmide) [inline src/pages/CatalogPage.tsx:50]: here u passing props but your query doesn't accept them

File: src/redux/api/product.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const productApi = createApi({
+  baseQuery: fetchBaseQuery({
+    baseUrl: 'https://dreamteam.onrender.com',
+  }),
+  endpoints: (builder) => ({
+    getProducts: builder.query({
+      query: () => `/products`,
+    }),
```
Comment (MaxSchmide) [inline src/redux/api/product.ts:10]: here you need accept those props and return the object like ({})

**Review by MaxSchmide — CHANGES_REQUESTED** [review]

**Review by MaxSchmide — APPROVED** [review]
Awesome

**Pull request merged**